### PR TITLE
📖 Brush up packaging.md

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR makes some improvements to packaging.md

## Related issue(s)

This continues the work on #1531 .

### CODE REVIEW

File: docs/content/v0.20/packaging.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+
+The following is a graph of the GitHub repositories and the dependencies among them. The repo at the tail of an arrow depends on the repo at the head of the arrow.
+
+```mermaid
+flowchart LR
+    kubestellar --> kubeflex
+    kubeflex    --> kubestellar
+    kubestellar --> ocm-status-addon
+    kubestellar --> ocm-transport-plugin
+    ocm-transport-plugin --> kubestellar
```
Comment (nirrozenbaum) [inline docs/content/v0.20/packaging.md:13]: why `kubestellar` repo depends on `ocm-transport-plugin` repo?

Comment (nirrozenbaum) [reply docs/content/v0.20/packaging.md:13]: as far as I know, `kubestellar` repo do not add reference to `ocm-transport-plugin` repo, and it shouldn't do so.
`ocm-transport-plugin` repo references `kubestellar` repo (in go.mod) cause it uses the generic transport controller (calls the cmd.Run) to initialize ocm based transport controller.

**Comment by nirrozenbaum** [comment]
left one question/comment. other than that, looks good to me.

File: docs/content/v0.20/packaging.md [hunk 2]
```diff
@@ -21,13 +63,15 @@ ends with `/github.com/kubestellar/kubestellar`.
 
 ### Derived files
 
-Some files in the kubestellar repo are derived from other files there. These are all generated by contributors and kept in GitHub. A contributor can use the individual `make` commands mentioned below to update subsets, or use the following command to update them all.
+Some files in the kubestellar repo are derived from other files there. These are all generated by contributors and kept in GitHub.
+
+Some of these derived files are derived by standard generators from the Kubernetes milieu. A contributor can use the following command to make all of those, or use the individual `make` commands described in the following subsubsections to update particular subsets.
```
Comment (ezrasilvera) [inline docs/content/v0.20/packaging.md:68]: the "some" implies we have files that are generated by none standard generators, what are those?

Comment (MikeSpreitzer) [reply docs/content/v0.20/packaging.md:67]: See the Helm chart section.

File: docs/content/v0.20/packaging.md (hunk 1 above)
Comment (MikeSpreitzer) [reply docs/content/v0.20/packaging.md:12]: Updated.

File: docs/content/v0.20/packaging.md [hunk 3]
```diff
@@ -21,13 +62,15 @@ ends with `/github.com/kubestellar/kubestellar`.
 
 ### Derived files
 
-Some files in the kubestellar repo are derived from other files there. These are all generated by contributors and kept in GitHub. A contributor can use the individual `make` commands mentioned below to update subsets, or use the following command to update them all.
+Some files in the kubestellar repo are derived from other files there. These are all generated by contributors and kept in GitHub.
```
Comment (ezrasilvera) [inline docs/content/v0.20/packaging.md:65]: "by contributors" is not clear to me . What does it mean ?  Everything in our repo is "generated by contributors" what is the difference ?

File: docs/content/v0.20/packaging.md [hunk 4]
```diff
@@ -21,13 +62,15 @@ ends with `/github.com/kubestellar/kubestellar`.
...
+
+Some of these derived files are derived by standard generators from the Kubernetes milieu. A contributor can use the following command to make all of those, or use the individual `make` commands described in the following subsubsections to update particular subsets.
 
 ```shell
 make all-generated
 ```
 
-The following command, checked in CI, checks whether all the derived files have been correctly derived. It must be invoked in a state where the `git status` is clean, or at least the dirty files are irrelevant; the current commit is what is checked.
+The following command, checked in CI, checks whether all those derived files have been correctly derived. It must be invoked in a state where the `git status` is clean, or at least the dirty files are irrelevant; the current commit is what is checked.
```
Comment (ezrasilvera) [inline docs/content/v0.20/packaging.md:73]: I thought it also generate them not just "check"

File: docs/content/v0.20/packaging.md (hunk 1 above)
Comment (nirrozenbaum) [reply docs/content/v0.20/packaging.md:12]: I was also doing a quick search now to find where `kubestellar` references `ocm-status-addon` and I didn't find such reference. 
I think only one direction exists (`ocm-status-addon` references `kubestellar`). this direction makes sense to me.

File: docs/content/v0.20/packaging.md (hunk 3 above)
Comment (MikeSpreitzer) [reply docs/content/v0.20/packaging.md]: Reworded.

File: docs/content/v0.20/packaging.md (hunk 4 above)
Comment (MikeSpreitzer) [reply docs/content/v0.20/packaging.md]: Noted.

File: docs/content/v0.20/packaging.md (hunk 1 above)
Comment (MikeSpreitzer) [reply docs/content/v0.20/packaging.md:14]: updated again.

**Pull request merged**